devsetup/scripts/standalone.sh

```diff
@@ -77,6 +79,16 @@ export NTP_SERVER=${NTP_SERVER:-"clock.corp.redhat.com"}
 export EDPM_COMPUTE_CEPH_ENABLED=${EDPM_COMPUTE_CEPH_ENABLED:-true}
 export CEPH_ARGS="${CEPH_ARGS:--e \$HOME/deployed_ceph.yaml -e /usr/share/openstack-tripleo-heat-templates/environments/cephadm/cephadm-rbd-only.yaml}"
 
+if [[ -f $HOME/containers-prepare-parameters.yaml ]]; then
+    echo "Using existing containers-prepare-parameters.yaml - contents:"
+    cat $HOME/containers-prepare-parameters.yaml
+else
+    openstack tripleo container image prepare default \
+        --output-env-file $HOME/containers-prepare-parameters.yaml
```

Comment:
the ceph command uses /root/containers-prepare-parameters.yaml as it runs as root user.

so will also need to copy it there after this generation

https://logserver.rdoproject.org/02/49902/1/check/data-plane-adoption-github-rdo-centos-9-crc-single-node/847f76c/ci-framework-data/logs/ci_make_002_run_standalone.log

The output file /home/zuul/containers-prepare-parameters.yaml can not be created. Error: [Errno 2] No such file or directory: '/home/zuul/containers-prepare-parameters.yaml'

Reply:
Also we need to escape `$HOME` because we need that to point to the the home of the vm and not of the host

Reply:
thanks @cescgina nice catch will add

Reply:
no actually, at this level we want it to pickup from the host.. it is doing the copy with ssh into the VM this is outside the heredoc

Reply:
you *are* right, i was looking at the other snippet below line 110  ... which is outside the heredoc

this one indeed is in the script that is executed on the vm ... adding \